apps/mark-scan/frontend/vite.config.ts

```diff
@@ -84,6 +84,10 @@ export default defineConfig((env) => {
       },
     ],
 
+    server: {
+      hmr: false,
```

Comment:
Why? If this is here does that other code that seems to disable it in JS also need to exist?

Reply:
I tried this first but couldn't get it working, so I added the code to disable in JS and forgot to remove this. But upon further review, this config on its own disables HMR, but the code change on its own doesn't. It's possible I forgot a server restart after adding this config, and only restarted once I added the code change. Regardless, I'm updating to keep this config and delete the lines that (unsuccessfully) disable it from code.

Reply:
[5ec62b4](https://github.com/votingworks/vxsuite/pull/3643/commits/5ec62b4b73a6161ae54e76aa014c57e588291cf3)